src/openforms/submissions/templates/submission_logs/submission_logs.html

```diff
@@ -0,0 +1,69 @@
+{% extends "admin/change_form.html" %}
+{% load i18n admin_urls static %}
+
+{% block extrastyle %}
+
+<link rel="stylesheet" href="{% static "bundles/core-css.css" %}">
+{{ block.super }}
+{% endblock %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+<a href="{% url 'admin:index' %}">{% translate 'Home' %}</a>
+&rsaquo; <a href="{% url opts|admin_urlname:'changelist' %}">{{ opts.verbose_name_plural|capfirst }}</a>
+&rsaquo; <a href="{% url 'admin:submissions_submission_change' original.pk %}">{{ original }}</a>
+&rsaquo;{% trans 'Evaluated Logic Logs'|capfirst%}
+</div>
+{% endblock %}
+
+{% block content %}
+    <h1>{% trans  "Logging" %}</h1>
+    {% block field_sets %}
+        {% for log in logs_activity  %}
+            <fieldset class="module aligned collapse collapsed">
+                <h2>{{ log.name|date:'Y-m-d H:i:s' }} - {% trans 'Logic' %} - </h2>
+
+                <h4> {% trans "Variables" %} </h4>
+                <table class="logic-rules-tables">
+                    <thead>
+                        <th class="property-name-width"> {% trans "Property name" %} </th>
+                        <th> {% trans "Value" %} </th>
+                    </thead>
+                    <tbody>
+                        
+                        {% for node in log.source_components  %}
+                            {% include "submission_logs/property_name.html" %}
+                        {% endfor %}
+                    </tbody>
+                </table>
+                <h4> {% trans "Logic rules" %} </h4>
+                <table class="logic-rules-tables">
+                    <thead>
+                        <th class="rule-width"> {% trans "Rule" %} </th>
+                        <th class="trigger-width"> {% trans "Trigger" %} </th>
+                        <th> {% trans "Action" %} </th>
+                    </thead>
+                    <tbody>
+                        {% for action in log.targeted_components %}
+                            <tr>
+                                <td>
+                                    {% for node in log.source_components  %}
+                                    {% include "submission_logs/rules.html" %}
+                                    {% endfor %}
+                                </td>
+                                <td>{% if log.trigger %}{% trans "Yes" %} {% else %} {% trans "No" %} {% endif %} </td>
```

Comment:
```suggestion
                                <td>{% if log.trigger %}{% trans "Yes" %} {% else %} {% trans "No" %} {% endif %} </td>
```
Look up the `yesno` template filter :smile: